dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -314,6 +317,34 @@ public void testComponent_linkageCheckResult() throws IOException, ParsingExcept
                 + " referenced from 21 classes ▶"); // '▶' is the toggle button
   }
 
+  @Test
+  public void testComponent_linkageCheckResult_java11() throws IOException, ParsingException {
+    String javaVersion = System.getProperty("java.version");
+    // javaMajorVersion is 1 when we use Java 8. Still good indicator to ensure Java 11 or higher.
+    int javaMajorVersion = Integer.parseInt(javaVersion.split("\\.")[0]);
+    Assume.assumeTrue(javaMajorVersion >= 11);
+
+    // version used in libraries-bom 1.0.0
+    Document document = parseOutputFile(
+        "com.google.http-client_google-http-client-appengine_1.29.1.html");
```

Comment:
Is this difference still present in more recent http clients? If not, we could simply update the version we use for the test.

If it is present, we should probably file a bug against the http client since it should work with Java 11.

Reply:
Added more description. This test does not check the difference between Java 8 and Java 11. Rather, this test ensures that the dashboard generated using Java 11 has the expected linkage error section.

For the linkage errors in google-http-client-appengine, the two artifacts ("commons-logging:commons-logging:1.2" and "com.google.appengine:appengine-api-1.0-sdk:1.9.71") still show the (known) linkage errors. They are present in Java 8 too: https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/snapshot/com.google.http-client_google-http-client-appengine_1.38.0.html